src/QtDBMigration.cpp

```diff
@@ -36,20 +37,27 @@ class QtDBMigrationPrivate
     bool migrate();
     bool migrate(int version);
 
+
 private:
     bool createMetaTable();
 
     QtDBMigration *m_q;
     AbstractConfig *m_config;
+    AbstractConfig *m_folder_config;
```

Comment:
No :).

The point is, that FolderConfig is set as m_config :).

Reply:
Ok, I think it should be fixed now :)